demo/index.html

```diff
@@ -32,13 +33,17 @@
   .wide {
     width: 474px;
   }
+  paper-spinner {
+    width: 16px;
+    height: 16px;
+  }
 </style>
 <body>
   <div class="horizontal center-center layout">
     <div>
       <h4>method="get"</h4>
       <div class="horizontal-section">
-        <form is="iron-form" id="formGet" method="get" action="/">
+        <form is="iron-form" id="formGet" method="get" action="https://httpbin.org/delay/2">
```

Comment:
I think I would prefer this being a separate demo. The `get`/`post` demos are super simple, and just try to demonstrate some elements and the basic submission. Maybe this could be labelled "Form with state changes" or something like that?